src/iomap.cpp

```diff
@@ -198,9 +198,22 @@ bool IOMap::loadMap(Map* map, const std::string& identifier)
 				setLastErrorString("Invalid map node.");
 				return false;
 			}
-
-			uint16_t base_x = area_coord.x;
-			uint16_t base_y = area_coord.y;
+			
+			uint16_t base_x = 0;
```

Comment:
if we take this branch:
```
if (place_x != 0 && place_y != 0) {
```
then:
```
base_x = place_x; 
base_y=place_y;
```
which means that all areas within a map file will be loaded into the same coordinates(overlayed on top of each other).